lighthouse-core/report/html/templates.html

```diff
@@ -769,9 +769,8 @@
 
           </span>
           <span class="crc-node__tree-value">
-            <span class="crc-node__tree-file"><!-- fill me: node.request.url.file --></span>
+            <a class="lh-link crc-node__tree-file" target="_blank"><!-- fill me: node.request.url.file --></a>
```

Comment:
obviously we do need to run lighthouse against itself, also need `noopener` here :)

Reply:
one day :)